doc/source/user_guide/groupby.rst

```diff
@@ -75,9 +75,9 @@ See the :ref:`cookbook<cookbook.grouping>` for some advanced strategies.
 Splitting an object into groups
 -------------------------------
 
-pandas objects can be split on any of their axes. The abstract definition of
-grouping is to provide a mapping of labels to group names. To create a GroupBy
-object (more on what the GroupBy object is later), you may do the following:
+The abstract definition of grouping is to provide a mapping of labels to
+group names. To create a GroupBy object (more on what the GroupBy object is
+later), you may do the following:
```

Comment:
`axis=1` hasn't been deprecate yet, but there'sa an open issue about (#51778). Can you wait with this section until that is merged.

Reply:
I can, but all the other references  to groupby axis=1, have already been removed in previous PR. It looks like this one was forgotten.
I think it should be removed.

Reply:
#51778 is about window/resample; groupby axis=1 has been deprecated in https://github.com/pandas-dev/pandas/issues/51203. I'm +1 on this change.